forch/forchestrator.py

```diff
@@ -136,6 +140,7 @@ def initialize(self):
         if self._calculate_config_files():
             self._initialize_faucetizer()
             self._faucetizer.reload_structural_config()
+            self._faucet_collector.set_faucetizer_enabled(True)
```

Comment:
This should be done as part of the initialization of  _faucet_collector, not dynamically later on. If necessary, you might need to separate out "calculate what should be enabled" from "create instances as necessary" -- right now they might be too bundled together.   E.g., move the base "self._calculate_config_files()" earlier and save to a local variable, and then use that various places as appropriate.

Reply:
Updated. PTAL..